src/main/archetype/all/src/main/content/META-INF/vault/filter.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<workspaceFilter version="1.0">
+    <filter root="/apps/${appsFolderName}-packages"/>
+    <filter root="/apps/vendor-packages"/>
```

Comment:
@DominikSuess WDYT about the convention of "vendor-packages" .. I think its safe since, IIUC, there should only be 1 "all" container package deployed ever. (Vs. having all-1 package installing something into /apps/vendor-packages, and all-2 package deploying its own.

Reply:
@davidjgonzalez , @vladbailescu   yep that might be a viable workaround to place the "all" packages of 3rd libraries in there considering these would be skyline compliant. That might eliminate the "trouble" with the core components all package (cc @bpauli ) while it still would be important that dependencies are not defined against the all packages but the concrete application & content packages.

Reply:
Updated to app-specific vendor packages: https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-bb796ff3d388b76d664d7349f45648b4R19